### PR TITLE
Add function getMessageInfo

### PR DESCRIPTION
**Pull request by shirser121**
Get the message info, delivery, reads etc...

### CODE REVIEW

**Comment by PurpShell** [comment]
did you know this actually exists?

https://docs.wwebjs.dev/Message.html#getInfo

File: src/Client.js [hunk 1]
```diff
@@ -641,6 +641,36 @@ class Client extends EventEmitter {
...
     }
 
+    /**
+     * Returns an object with information about the message
+     * @param {string} messageId
+     * @returns {Promise<object>} message information
+     */
+    async getMessageInfo(messageId) {
+        return await this.pupPage.evaluate(async messageId => {
+            const msgExist = await window.Store.MsgInfo._find(messageId);
```
Comment (PurpShell) [inline src/Client.js:651]: you seem to be missing some stuff in Injected.js, or is this something that we already put to Store?

Comment (shirser121) [reply src/Client.js:651]: Please explain the question.
MsgInfo is already exist

**Comment by shirser121** [comment]
> did you know this actually exists?
> 
> https://docs.wwebjs.dev/Message.html#getInfo

I just want to improve that and add it to client

File: src/Client.js (hunk 1 above)
Comment (ceringo) [reply src/Client.js:651]: is this method working?

**Pull request closed without merging**